mentorship ios/ViewModels/LoginViewModel.swift

```diff
@@ -11,9 +11,7 @@ class LoginViewModel: ObservableObject {
     
     // MARK: - Variables
     @Published var loginData = LoginModel.LoginUploadData(username: "", password: "")
-    @Published var loginResponseData = LoginModel.LoginResponseData(message: "", accessToken: "")
-    @Published var inActivity: Bool = false
-    private var cancellable: AnyCancellable?
+    @Published var loginResponseData = LoginModel.LoginResponseData(message: "")
```

Comment:
are we displaying all informations from LoginResoinseData and LoginUploadData in exact form as it is here?

Reply:
yes